src/components/Reports/GoalCalculator/CalculatorSettings/Steps/InformationStep/InformationStep.tsx

```diff
@@ -175,47 +219,177 @@ export const InformationStep: React.FC<InformationStepProps> = () => {
+    // Financial form initial values for spouse
+    paycheckAmount: 0,
+    taxes: 0,
+    secaStatus: '',
+    contributionRoth403b: 0,
+    contributionTraditional403b: 0,
+    mhaAmountPerPaycheck: 0,
+
+    // Personal form initial values for spouse
+    role: '',
+    tenure: '',
+    age: '',
+  };
+
   const handleSubmit = () => {
     // Handle form submission here
     // TODO: Implement form submission logic
 
     handleContinue();
   };
 
+  // Someone may or may not have a spouse,
+  // set to null until query when we have real data
+  const [spouseInformation, setSpouseInformation] = useState<boolean | null>(
+    false,
+  );
+  const [buttonText, setButtonText] = useState<string>(t('View Spouse'));
+
+  const onClickSpouseInformation = () => {
+    setSpouseInformation(!spouseInformation);
+    // Change 'Spouse' here to be actual spouse's name if available
+    setButtonText(
+      spouseInformation ? t('View Spouse') : t('View Your Information'),
+    );
+  };
+
   return (
     <>
-      <StyledTypography>
+      <StyledTypography data-testid="verify-info-typography">
         {t('Take a moment to verify your information.')}
       </StyledTypography>
-      <Formik
-        initialValues={initialValues}
-        validationSchema={validationSchema}
-        onSubmit={handleSubmit}
-        enableReinitialize
-      >
-        <Form>
-          <StyledCard>
-            <StyledInfoBox>
-              <StyledTabs
-                value={value}
-                onChange={handleChange}
-                aria-label={t('information tabs')}
-              >
-                <Tab label={t('Personal')} />
-                <Tab label={t('Financial')} />
-              </StyledTabs>
-            </StyledInfoBox>
-
-            <TabPanel value={value} index={0}>
-              <InformationStepPersonalForm />
-            </TabPanel>
-
-            <TabPanel value={value} index={1}>
-              <InformationStepFinancialForm />
-            </TabPanel>
-          </StyledCard>
-        </Form>
-      </Formik>
+      <StyledCard>
+        <Box
+          display="flex"
+          gap={2}
+          m={2}
+          alignItems="center"
+          justifyContent="space-between"
+        >
+          <Box
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            gap={1}
+          >
+            {userData?.user ? (
+              <Avatar
+                src={userData.user.avatar}
+                alt={userData.user.firstName ?? t('User')}
+                variant="rounded"
+                sx={{ width: 36, height: 36, marginRight: 1 }}
+              />
+            ) : (
+              <StyledPersonBox>
+                <PersonIcon
+                  sx={{ fontSize: 40, color: theme.palette.primary.main }}
+                />
+              </StyledPersonBox>
+            )}
+            <Typography data-testid="info-name-typography" align="center">
```

Comment:
The alignment doesn't seem to do anything.

```suggestion
            <Typography data-testid="info-name-typography">
```